docs/questions/query-builder/expressions/converttimezone.md

```diff
@@ -0,0 +1,139 @@
+---
+title: ConvertTimezone
+---
+
+# ConvertTimezone
+
+`convertTimezone` assigns a new time zone to a datetime value (by adding to, or subtracting from the original datetime).
+
+| Syntax                                                                                                    | Example                                                       |
+|-----------------------------------------------------------------------------------------------------------|---------------------------------------------------------------|
+| `convertTimezone(column, target, source)`                                                                 | `convertTimezone("December 28, 2022, 12:00:00", "EST, "PST")` |
+| Assigns the target time zone to your datetime column. You can optionally specify a source time zone too.  | `December 28, 2022, 9:00:00`                                  |
```

Comment:
what do you mean by assign? 
My understanding of assign is like appending to a string without changing anything.

If that's the case then we don't simply just assign the timezone, but we actually `shift` the time so that the input column is displayed in the `target-timezone`